Formula/or-tools.rb

```diff
@@ -0,0 +1,60 @@
+class OrTools < Formula
+  desc "Google's Operations Research tools"
+  homepage "https://developers.google.com/optimization/"
+  url "https://github.com/google/or-tools/archive/v8.0.tar.gz"
+  sha256 "ac01d7ebde157daaeb0e21ce54923a48e4f1d21faebd0b08a54979f150f909ee"
+  license "Apache-2.0"
+  head "https://github.com/google/or-tools/archive/master.zip"
+
+  bottle do
+    cellar :any
+  end
+
+  depends_on "cmake" => :build
+  depends_on "pkg-config" => :build
+  depends_on "abseil"
+  depends_on "cbc"
+  depends_on "cgl"
+  depends_on "clp"
+  depends_on "coinutils"
+  depends_on "gflags"
+  depends_on "glog"
+  depends_on "osi"
+  depends_on "protobuf"
+
+  # This is a CMake based build
```

Comment:
```suggestion
```

Reply:
Applied your patch
side note: For or-tools we maintain 3 build system (makefile (legacy), CMake (FOSS friendly IMHO), bazel (oss version of the google build system use internally)